<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-03-07T11:45:21 WAT" name="Suite" finished-at="2022-03-07T11:46:47 WAT" duration-ms="86258">
    <groups>
    </groups>
    <test started-at="2022-03-07T11:45:21 WAT" name="EFMS Automation Test Execution" finished-at="2022-03-07T11:46:47 WAT" duration-ms="86258">
      <class name="main.Checkers">
        <test-method signature="ViewCases()[pri:0, instance:main.Checkers@47e2e487]" started-at="2022-03-07T11:46:22 WAT" name="ViewCases" finished-at="2022-03-07T11:46:47 WAT" duration-ms="25512" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [WAKILU ALAYO LUKMAN] but found [WAKILU ALAYO LUKMAN
6290449690]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [WAKILU ALAYO LUKMAN] but found [WAKILU ALAYO LUKMAN
6290449690]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at main.Checkers.ViewCases(Checkers.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ViewCases -->
      </class> <!-- main.Checkers -->
      <class name="main.authentication">
        <test-method signature="loginWithValidCredentials()[pri:0, instance:main.authentication@51e69659]" started-at="2022-03-07T11:45:58 WAT" name="loginWithValidCredentials" finished-at="2022-03-07T11:46:22 WAT" duration-ms="24135" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginWithValidCredentials -->
      </class> <!-- main.authentication -->
      <class name="main.pC">
        <test-method is-config="true" signature="launchbrowser()[pri:0, instance:main.pC@32502377]" started-at="2022-03-07T11:45:21 WAT" name="launchbrowser" finished-at="2022-03-07T11:45:58 WAT" duration-ms="36589" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchbrowser -->
      </class> <!-- main.pC -->
    </test> <!-- EFMS Automation Test Execution -->
  </suite> <!-- Suite -->
</testng-results>
